vendor/github.com/ONSdigital/go-ns/kafka/consumer-group.go

```diff
@@ -5,9 +5,10 @@ import (
 	"time"
 
 	"context"
+	"errors"
```

Comment:
Is this PR missing the vendor.json file? As `govendor status` throws the following error; 

`The following packages are missing or modified locally:
	github.com/ONSdigital/go-ns/kafka
Error: status failed for 1 package(s)`